test/integration/host_test.rb

```diff
@@ -448,18 +445,18 @@ def test_show_action_buttons
       visit edit_host_path(host)
       assert page.has_link?('Parameters', :href => '#params')
       click_link 'Parameters'
-      assert page.has_selector?('#inherited_parameters .btn[data-tag=override]')
+      assert_selector('#inherited_parameters .btn[data-tag=override]')
       page.find('#inherited_parameters .btn[data-tag=override]').click
-      assert page.has_no_selector?('#inherited_parameters .btn[data-tag=override]')
+      refute_selector('#inherited_parameters .btn[data-tag=override]')
```

Comment:
What's more specific about it?  Is it more specific than `page` - some part of the DOM?  I'm not sure I see what's different.